docker-compose.yaml

```diff
@@ -13,22 +13,33 @@ services:
     environment:
       PHP_IDE_CONFIG: serverName=php-cs-fixer
       PHP_CS_FIXER_ALLOW_XDEBUG: 1
+
   php-8.0:
     <<: *php
     build:
       args:
         ALPINE_VERSION: "3.16"
         PHP_VERSION: "8.0"
         PHP_XDEBUG_VERSION: "3.2.2"
+
   php-8.1:
     <<: *php
     build:
       args:
         PHP_VERSION: "8.1"
         PHP_XDEBUG_VERSION: "3.2.2"
+
   php-8.2:
     <<: *php
     build:
       args:
         PHP_VERSION: "8.2"
         PHP_XDEBUG_VERSION: "3.2.2"
+
+  php-8.3:
+    <<: *php
+    build:
+      args:
+        ALPINE_VERSION: "3.17"
```

Comment:
do we need to define explicit apline version? and why here, but not in others?

Reply:
Good catch, I don't remember why it was there (I had this changes stashed for few days), but it wasn't needed here. `ALPINE_VERSION` is defined in `Dockerfile` and defaults to 3.18, which should be used for PHP 8.3. There is lower Alpine version set for 7.4 and 8.0.

Reply:
yeah, I remember we needed lower version for low PHP, but we do not need to lower it for 8.3 ;)